platforms/common/src/CommonKernels.cpp

```diff
@@ -2761,8 +2758,8 @@ void CommonCalcCustomGBForceKernel::initialize(const System& system, const Custo
         globals.initialize<float>(cc, force.getNumGlobalParameters(), "customGBGlobals");
     vector<vector<float> > paramVector(paddedNumParticles, vector<float>(numParams, 0));
     vector<vector<int> > exclusionList(numParticles);
+    vector<double> parameters;
```

Comment:
I think moving this out of the block will create a naming conflict.  Later in the same method (line 3438) we declare another variable called `parameters`.

Reply:
I've changed this back (and all of the other changes to the constructors). The only changes now are the `thread_local statics` added to the ForceInfo methods.